terraform/parser.go

```diff
@@ -48,7 +48,7 @@ func NewDefaultParser() *DefaultParser {
 // NewFmtParser is FmtParser initialized with its Regexp
 func NewFmtParser() *FmtParser {
 	return &FmtParser{
-		Fail: regexp.MustCompile(`(?m)^(diff a/)`),
+		Fail: regexp.MustCompile(`(?m)^@@[^@]+@@`),
```

Comment:
If someone uses terraform v0.11.x, is this case failed, right? (Of course, this is not a problem for v0.12.x).

Reply:
Right.

There are several variants for [Unified format](https://en.wikipedia.org/wiki/Diff#Unified_format), but the format of the range information line is alway as follows:

```
@@ -l,s +l,s @@ optional section heading
```

``` diff
This lines are the header of hunk.
It may contain extra information for other tools (e.g. git, svn).
But sometimes there is no line in the header.
versions.tf
--- old/versions.tf
+++ new/versions.tf
@@ -1,4 +1,4 @@
 
 terraform {
-  required_version     = ">= 0.12"
+  required_version = ">= 0.12"
 }
```

Reply:
👀

Reply:
Hi
What should I do to merge this pull request?

Reply:
Sorry for the late reply...! I'll test this patch in my local and gonna merge soon.

Reply:
Confirmed this worked in my local also and was compatible with current version.